src/ext/java/org/opentripplanner/ext/siri/SiriTimetableSnapshotSource.java

```diff
@@ -612,6 +613,12 @@ private Result<Void, UpdateError> handleAddedTrip(
     var tripBuilder = Trip.of(tripId);
     tripBuilder.withRoute(route);
 
+    if (!route.getMode().equals(transitMode.first)) {
```

Comment:
We don't need this, as it is only set in the constructor if different from the route